Plugins/NativeEngine/Source/NativeEngine.cpp

```diff
@@ -1880,6 +1882,9 @@ namespace Babylon
         }
 
         Graphics::FrameBuffer* frameBuffer = new Graphics::FrameBuffer(m_deviceContext, frameBufferHandle, width, height, false, generateDepth, generateStencilBuffer);
+        // Clear framebuffer : https://registry.khronos.org/webgl/specs/latest/1.0/#TEXIMAGE2D
+        auto encoder = GetUpdateToken().GetEncoder();
```

Comment:
This change looks fishy to me. Can we discuss offline?

Reply:
Discussed offline.